pkg/readiness/object_tracker.go

```diff
@@ -29,13 +30,16 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 )
 
+var readinessRetries = flag.Int("readiness-retries", 0, "The number of resource ingestion attempts allowed before the resource is disregarded.  A value of -1 will retry indefinitely.")
```

Comment:
Can you please add this flag to the readme so people know how to use it?

Reply:
@ritazh I've added some information to the README.  Could you take a look?

Reply:
Thanks @julianKatz!